config.js

```diff
@@ -822,6 +822,8 @@ config.NSFS_UPDATE_ISSUES_REPORT_ENABLED = true;
 config.NSFS_EXIT_EVENTS_TIME_FRAME_MIN = 24 * 60; // per day
 config.NSFS_MAX_EXIT_EVENTS_PER_TIME_FRAME = 10; // allow max 10 failed forks per day
```

Comment:
please update NonContainerizedDeveloperCustomizations.md file with this new config and maybe also the doc where you wrote about supplemental groups last PR